src/platform/qt/Window.cpp

```diff
@@ -123,9 +125,10 @@ Window::Window(ConfigController* config, int playerId, QWidget* parent)
 	}, this);
 	m_config->updateOption("showLibrary");
 
-	connect(m_libraryView, &LibraryView::accepted, [this]() {
+	connect(m_libraryView, &LibraryController::startGame, [this]() {
 		VFile* output = m_libraryView->selectedVFile();
-		QPair<QString, QString> path = m_libraryView->selectedPath();
+		mLibraryEntry* entry = m_libraryView->selectedGame();
+		QPair<QString, QString> path = qMakePair(entry->filename, entry->base);
```

Comment:
paging @endrift: Should there even be a `selectedVFile` method? Or should that be taken care of here, since it fetches the Entry anyway?

Also what terminology should I use? "selectedGame" was the name in Dolphin, maybe "selectedEntry" is more appropriate here?